pkg/operator/ceph/file/filesystem.go

```diff
@@ -189,33 +189,58 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 		return fmt.Errorf("Cannot create multiple filesystems. Enable %s env variable to create more than one", client.MultiFsEnv)
 	}
 
-	logger.Infof("Creating filesystem %s", f.Name)
-	err = client.CreatePoolWithProfile(context, clusterName, *f.metadataPool, appName)
+	poolNames, err := client.GetPoolNamesByID(context, clusterName)
 	if err != nil {
-		return fmt.Errorf("failed to create metadata pool '%s': %+v", f.metadataPool.Name, err)
+		return fmt.Errorf("failed to get pool names. %+v", err)
+	}
+
+	logger.Infof("Creating filesystem %s", f.Name)
+
+	// Make easy to locate a pool by name and avoid to repeat searchs
+	reversed_pool_map := make(map[string]int)
```

Comment:
nit: camel casing is preferred. So we might have `reversedPoolMap`, `poolFound`, `poolsCreated`, etc

Reply:
Got it!. thx!